packages/rich-list-core/src/types.ts

```diff
@@ -1,10 +1,9 @@
-import { AccountAmount } from '@defichain/jellyfish-api-core/src/category/account'
+import { BigNumber } from '@defichain/jellyfish-api-core'
 
-export interface ActiveAddressAccountAmount {
-  [key: string]: AccountAmount
+export interface AccountAmount {
+  [id: string]: BigNumber
 }
 
-export interface AddressBalance {
-  address: string
-  amount: number
+export interface ActiveAddressAccountAmount {
+  [key: string]: AccountAmount
```

Comment:
I think we can move all this into `RichListCore.ts` since its the only file that uses these data structure.